tests/prague/eip7702_set_code_tx/test_gas.py

```diff
@@ -435,6 +435,14 @@ def gas_test_parameter_args(
             },
             id="single_invalid_authorization_invalid_chain_id_single_signer",
         ),
+        pytest.param(
+            {
+                "authority_type": AddressType.EOA_WITH_SET_CODE,
```

Comment:
Just looking now and had a quick q. I noticed this is not in the `if include_pre_authorized:` test cases. `EOA_WITH_SET_CODE` implies pre-authorized, right? If so, should this be `AddressType.EOA`? This passes all the tests but I want to make sure that's your intended test case.

Reply:
nm, I wasn't looking at the code block in the description. I'll keep that in mind 😅.